packages/trpc/server/routers/loggedInViewer/addNotificationsSubscription.handler.ts

```diff
@@ -0,0 +1,30 @@
+import prisma from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TAddNotificationsSubscriptionInputSchema } from "./addNotificationsSubscription.schema";
+
+type AddSecondaryEmailOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TAddNotificationsSubscriptionInputSchema;
+};
+
+export const addNotificationsSubscriptionHandler = async ({ ctx, input }: AddSecondaryEmailOptions) => {
+  const { user } = ctx;
+  const { subscription } = input;
+
+  const existingSubscription = await prisma.notificationsSubscriptions.findFirst({
+    where: { userId: user.id, subscription },
+  });
+
+  if (!existingSubscription) {
+    await prisma.notificationsSubscriptions.create({
+      data: { userId: user.id, subscription },
+    });
+  }
```

Comment:
I think we need some integration tests for the backend changes. What do you all think? @keithwillcode @Udit-takkar

Reply:
I don't think so. It's just simply adding and removing from db and not doing anything complex yet.